scripts/test_ports/vcpkg-ci-nanobind/portfile.cmake

```
@@ -0,0 +1,23 @@
+set(VCPKG_POLICY_EMPTY_PACKAGE enabled)
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO wjakob/nanobind_example
+    REF cc116d7c87d2e19c9c8146464c6ae7ed17620eec
+    SHA512 cdb0eb09b1c03c0dea291daf876a85f9d5641f57747786cd2289d0aa4c8e3f34bd2809c351b3231fb80a358615086ee0e687ce23999a9ae012f75b000bdeef10
+    HEAD_REF master
+)
+
+# This is needed to correctly build/link against a debug build of Python on
+# Windows
+set(VCPKG_CXX_FLAGS_DEBUG -DPy_DEBUG ${VCPKG_CXX_FLAGS_DEBUG})
+set(VCPKG_C_FLAGS_DEBUG -DPy_DEBUG ${VCPKG_C_FLAGS_DEBUG})
```

Note that these are strings (as on the command line), not CMake lists.
```suggestion
string(APPEND VCPKG_C_FLAGS_DEBUG " -DPy_DEBUG")
string(APPEND VCPKG_CXX_FLAGS_DEBUG " -DPy_DEBUG")
```

---

Thank you!